src/infrastructure/applications/api.js

```diff
@@ -1,14 +1,13 @@
-const rp = require('login.dfe.request-promise-retry');
+const { fetchApi } = require('login.dfe.async-retry');
 const jwtStrategy = require('login.dfe.jwt-strategies');
 const config = require('../config');
 
 const callApi = async (endpoint, method, body, correlationId) => {
   const token = await jwtStrategy(config.applications.service).getBearerToken();
 
   try {
-    return await rp({
+    return await fetchApi(`${config.applications.service.url}/${endpoint}`,{
```

Comment:
`strictSSL` also isn't used; this was a feature of `request-promise` which is now handled more consistently with the built-in `NODE_TLS_REJECT_UNAUTHORIZED` feature of node for local testing.

Reply:
Updated